src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -144,7 +144,7 @@ public function load(array $configs, ContainerBuilder $container): void
 
                 break;
             case 'sonata.admin.security.handler.acl':
-                if (!$container->has('security.acl.provider')) {
+                if (!$container->hasParameter('security.acl.provider')) {
```

Comment:
why don't we instead check for the presence of the bundle enabled?, something like:

```suggestion
                if (!isset($bundles['AclBundle'])) {
```

Reply:
that actually might work better. I was checking bundles in compiler passes and extensions and it never came to my mind to simply replace this check with bundle check, although I wonder if that's the only bundle which can do ACL. I'll add it in front and I guess I can leave this as fallback since it should work with simple or statement.

Reply:
Keep in mind that the only integration we support is that bundle, We use some interfaces from there and dependencies also declared there. I wouldn't compplicate it any further, to me we should remove other checks and leave only the bundle one, wdyt @VincentLanglet ?

Reply:
I thought this might be the case. If support is only with `symfony/acl-bundle`, then bundle check and error message change should be enough - if it only checks if bundle exists, then the message should probably not mention `security.acl.provider` at all, just the part with required bundle.